tst/EnergyPlus/unit/EvaporativeCoolers.unit.cc

```diff
@@ -930,7 +930,8 @@ TEST_F(EnergyPlusFixture, EvapCoolerAirLoopPumpCycling)
 
     // Simulate air loop component calls SimEvapCooler
     // SimEvapCooler calls InitEvapCooler(EvapCoolNum) and CalcDirectEvapCooler
-    SimAirServingZones::SimAirLoopComponent(*state, EvapCond(EvapCoolNum).EvapCoolerName, Evap_Cooler_CompType, false, AirLoopNum, EvapCoolNum, 0);
+    SimAirServingZones::SimAirLoopComponent(
+        *state, EvapCond(EvapCoolNum).EvapCoolerName, Evap_Cooler_CompType, false, AirLoopNum, EvapCoolNum, 0, 0, 0, 0);
```

Comment:
Modifies existing unit test by adding default values for the three new arguments added for SimAirLoopComponent().